### PR TITLE
Task/add flag to enable erc20token

### PR DESCRIPTION
**Pull request by bailantaotao**

### CODE REVIEW

File: cmd/indexer/root.go [hunk 1]
```diff
@@ -180,9 +175,12 @@ func init() {
...
 		// Profling flags
-		ServerCmd.Flags().BoolVar(&profiling, "pprof", false, "Enable the pprof HTTP server")
-		ServerCmd.Flags().IntVar(&profilPort, "pprof.port", 8000, "pprof HTTP server listening port")
-		ServerCmd.Flags().StringVar(&profilHost, "pprof.host", "0.0.0.0", "pprof HTTP server listening interface")
+		ServerCmd.Flags().BoolVar(&profiling, flags.PprofEnable, false, "Enable the pprof HTTP server")
+		ServerCmd.Flags().IntVar(&profilPort, flags.PprofPort, 8000, "pprof HTTP server listening port")
+		ServerCmd.Flags().StringVar(&profilHost, flags.PprofHost, "0.0.0.0", "pprof HTTP server listening interface")
+
+		// erc20 flags
+		ServerCmd.Flags().BoolVar(&subscribeErc20token, flags.SubscribeErc20token, false, "Enable subscribe erc20 token")
```
Comment (jeddie-yhc) [inline cmd/indexer/root.go]: can we add some words in the description as well?
```
Enable erc20 token subscription. Please specify the erc20 tokens in configs/erc20.yaml
```

Comment (bailantaotao) [reply cmd/indexer/root.go]: OK

File: cmd/indexer/root.go [hunk 2]
```diff
@@ -131,6 +125,7 @@ var ServerCmd = &cobra.Command{
 			}()
 		}
 
+		log.Info("Starting eth-indexer")
```
Comment (shad0356jc) [inline cmd/indexer/root.go]: I know why you add this log lol.

Comment (bailantaotao) [reply cmd/indexer/root.go]: :laughing: :laughing: :laughing:

**Pull request merged**